.github/workflows/master-build.yml

```diff
@@ -0,0 +1,32 @@
+name: master build
+on:
+  push:
+    branches:
+      - master
+jobs:
+  validate:
+    name: Validate
+    runs-on: ubuntu-latest
+    container: kedacore/build-tools:latest
```

Comment:
this buildtools contains operator-sdk v0.12.0 https://github.com/kedacore/keda/blob/master/tools/build-tools.Dockerfile#L56

we should the one from the v2 branch which has operator-sdk v0.17.0, eg: https://github.com/kedacore/keda/blob/v2/.github/workflows/pr-validation.yml#L8